### ISSUE TITLE
Unknown mmp subkey: $AREA

### ISSUE THREAD

**Issue by MuellerSeb** [ufz/ogs5-benchmarks#27]
I found an unknown Subkey-words (`$AREA`) in the following two mmp files:

    T/HT_var_density_1D.mmp
    T/Viscosity/viscosity_yaws.mmp

Should it be `$GEOMETRY_AREA`?

**Comment by MuellerSeb** [comment]
https://github.com/GeoStat-Framework/ogs5py_benchmarks/blob/master/benchmarks_FEM_active/LOG.txt#L154
https://github.com/GeoStat-Framework/ogs5py_benchmarks/blob/master/benchmarks_FEM_active/LOG.txt#L166

**Comment by wenqing** [comment]
@MuellerSeb  You are right. For these two benchmarks, the variable corresponding to $GEOMETRY_AREA takes the default value of 1.0  because that the wrong keyword is skipped in the reading. 

Thanks for the findings. Would you please send a PR for this?
